gulpfile.babel.ts

```diff
@@ -203,11 +203,14 @@ async function browserifyTask(options: BrowserifyTaskOptions): Promise<void> {
       filename: destName,
       ...(options.standalone
         ? {
-            libraryTarget: 'umd',
-            libraryExport: 'default',
-            library: options.standalone,
+            library: {
+              export: 'default',
+              name: options.standalone,
+              type: 'window',
+            },
```

Comment:
What's the effect of this change?

Reply:
I was thinking about this config for npm usages this morning, and this's probably not what we want for all four of the current build targets this PR touches (npm, remote, integratedPageWorld, and test).

`integratedPageWorld`, `remote`, and `test` (for now) builds should be using `library.type` `window` instead of UMD imo.

We could have UMD as the output format still for npm. Browserify was doing 

```
--standalone -s    Generate a UMD bundle for the supplied export name.
                   This bundle works with other module systems and sets the name
                   given as a window global if no module system is found.
```

That said, ESM has been de rigueur for output for some time. I'd prefer to output ESM (ES 2020) for our npm builds. 

For another library's perspective, `uuid` v9's changelog:

> Remove the minified UMD build from the package.
> 
> Minified code is hard to audit and since this is a widely used library it seems more appropriate nowadays to optimize for auditability than to ship a legacy module format that, at best, serves educational purposes nowadays.
> 
> For production browser use cases, users should be using a bundler. For educational purposes, today's online sandboxes like replit.com offer convenient ways to load npm modules, so the use case for UMD through repos like UNPKG or jsDelivr has largely vanished.

Reply:
Maintaining the UMD version for `npm` output until we make the breaking change of ESM output, I think, is the way to go for now. Whether or not we want `UMD` for `remote` is an open question.